ion-hash/src/representation.rs

```diff
@@ -7,10 +7,10 @@
 //! and not speed.
 
 use ion_rs::{
+    binary::writer::raw,
```

Comment:
There are already some modules for encoding primitive types like `VarUInt`, `Int`, etc. This probably belongs with those? You could make the case that those should be moved to this module.

Reply:
agreed